src/mongocxx/options/server_api.cpp

```diff
@@ -29,15 +32,15 @@ std::string server_api::version_to_string(server_api::version version) {
         case server_api::version::k_version_1:
             return "1";
         default:
-            throw std::logic_error{"invalid server API version"};
+            throw mongocxx::logic_error { mongocxx::error_code::k_invalid_parameter, "invalid server API version" };
```

Comment:
Please run `python ./etc/clang_format.py format` to format. This should resolve the [lint](https://spruce.mongodb.com/task/cxx_driver_ubuntu1804_release_latest_lint_patch_8f25e2348f1db555cdea8a56036a7563b0817e02_61033720e3c33110c85e91e4_21_07_29_23_27_14/logs?execution=0) task on Evergreen (which was failing on a prior commit).

Reply:
+1, done.